src/main/java/Esteban.java

```diff
@@ -0,0 +1,139 @@
+                    Data.write(tasks);
+                } catch (StringIndexOutOfBoundsException e) {
+                    System.out.println("  (!) Invalid/missing values");
+                    System.out.println("  (!) Format: /deadline <description> -by <date>");
+                }
+
+                break;
+            case EVENT:
+                try {
+                    String des = line.substring(7, line.indexOf("-from")).strip();
+                    String from = line.substring(line.indexOf("-from") + 6, line.indexOf("-to")).strip();
+                    String to = line.substring(line.indexOf("-to") + 4).strip();
+                    Event newEvent = new Event(des, from , to, false);
+                    tasks.add(newEvent);
+                    System.out.println(" (+) Added: "+ newEvent);
+                    System.out.println(" (!) You have " + tasks.size() + " tasks in the list");
+                    Data.write(tasks);
+                } catch (StringIndexOutOfBoundsException e) {
+                    System.out.println("  (!) Invalid/missing values");
+                    System.out.println("  (!) Format: /event <description> -from <date> -to <date>");
+                }
+                break;
+            case DONE:
+                try {
+                    String[] values = line.split(" ");
+                    int value = Integer.parseInt(values[1]);
+                    tasks.get(value - 1).setStatus(true);
+                    System.out.println(" (+) Marked as Done: "+ tasks.get(value - 1).toString());
+                    Data.write(tasks);
+                } catch (ArrayIndexOutOfBoundsException e) {
+                    System.out.println("  (!) Task ID cannot be empty!");
+                    System.out.println("  (!) Format: /done <id>");
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("  (!) Task ID does not exist >:(");
+                }
+                break;
+            case DELETE:
+                try {
+                    String[] values = line.split(" ");
+                    int value = Integer.parseInt(values[1]);
+                    Task currentTask = tasks.get(value - 1);
+                    tasks.remove(currentTask);
+                    System.out.println("  (-) Removed: "+ currentTask.toString());
+                    System.out.println("  (!) You have " + tasks.size() + " tasks in the list");
+                    Data.write(tasks);
+                } catch (ArrayIndexOutOfBoundsException e) {
+                    System.out.println("  (!) Task ID cannot be empty!");
+                    System.out.println("  (!) Format: /delete <id>");
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("  (!) Task ID does not exist >:(");
+                }
+                break;
+            default:
+                System.out.println("  (!) Unrecognised Command! ");
+            }
+        }
+    }
```

Comment:
You should minimize the size of the main method by splitting the different functionalities into their own methods. By reducing the size of the method and ensuring each method does one functionality it is supposed to do, your code is easier to maintain, scale and debug.